IBM Resilient Systems- changed type of api fields (#11962)
demisto/content: Packs/IBMResilientSystems/ReleaseNotes/1_0_5.md

#### Integrations
##### IBM Resilient Systems

> - Fixed an issue where 'password' field was shown when entering API secrets while configuring instance.

Reviewer: ```suggestion
- Fixed an issue where the *password* parameter was not encrypted in the logs.
```
